BackendTaskPilot/appsettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
```

Comment:
Change Default into Warning in case of we want to sure that we are able to log warnings into prod environment.